docs/cpan-dependency-confusion.md

```diff
@@ -225,10 +225,19 @@ Use reserved namespaces that can't resolve to public things:
 
 From [PAUSE Naming Modules - Local::](https://pause.perl.org/pause/query?ACTION=pause_namingmodules#Local) and [The Perl Module Library (perlmodlib)](https://perldoc.perl.org/perlmodlib#Guidelines-for-Module-Creation)
 
+#### Namespace vacuum
+The CPAN ecosystem is not vulnerable to "Namespace vacuum" as described in [this article](https://cybersecuritynews.com/popular-python-library-vulnerability/) (python/PyPI).
+
+Indeed for the following reasons: 
+1. Removing all releases of a namespace does not release permissions
+2. Giving up ownership (release "FIRSTCOME permissions") does not release a name to the public
```

Comment:
Could we use a different word for "release" (meaning, "give up") in point 2, since the word "release" in point 1 talks about something entirely different ("published versions of a distribution")?

e.g. use "relinquish", OR rewrite point 2 to this:

2. Giving up "FIRSTCOME permissions" (ownership) does not release the name to the public.

Reply:
Very good point, I edited and even went a little further than you